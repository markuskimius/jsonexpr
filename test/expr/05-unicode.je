mystr = "\x21\u0394\U0001f6ab";
PRINT(mystr);
PRINT(mystr + " is " + LEN(mystr) + " bytes long.");
PRINT(mystr + " is " + STRLEN(mystr) + " characters long.");
