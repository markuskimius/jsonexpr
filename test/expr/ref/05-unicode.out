!Î”ğŸš«
!Î”ğŸš« is 7 bytes long.
!Î”ğŸš« is 3 characters long.
